### PR TITLE
Check-in Security Fixes

### PR DESCRIPTION
**Pull request by dev-zaid**
# Description
- Delete user email from check-in channel after check-in.
- Update database with user details after successful check-in.

## Type of change
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?
Discord

# Screenshot
image
image

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by dev-zaid** [comment]
Database task yet to be done.

File: src/service/checkin-service.ts [hunk 1]
```diff
@@ -120,7 +143,19 @@ const checkInEmails = async (
                 if (event.checkin?.teamEvent) {
                     await findAndJoinTeams(incomingMessage, registrant, event);
                 }
-                incomingMessage.react(CONSTANTS.checkinReactions.accept);
+                await incomingMessage.delete();
+                await dbCheckin.insertOne(<checkInDBSchema>{
+                    name: registrant.name,
+                    discordID: parseInt(incomingMessage.author.id),
```
Comment (shawshankkumar) [inline src/service/checkin-service.ts:149]: Store it as a string. No need to parse.

**Review by shawshankkumar — COMMENTED** [review]
Change your prettier settings to match Jack's. There are many unnecessary changes in the indentation which makes it hard to review the necessary changes.

File: src/service/checkin-service.ts [hunk 2]
```diff
@@ -93,15 +98,32 @@ const checkInEmails = async (
     try {
         const event = await getEvent(eventSlug, "checkin");
         if (!event) throw "eventKey Not Found in NodeCache!";
+        const ledgerChannel = (await incomingMessage.client.channels.fetch(
```
Comment (shawshankkumar) [inline src/service/checkin-service.ts:101]: What is the use of const ledgerChannel?

**Review by shawshankkumar — COMMENTED** [review]
Make a new branch after checking out from main, and make sure you only change what you want, not indentations and spacing and line changes.

**Comment by shawshankkumar** [comment]
Make a new branch, after checking out from master, this pr is being rejected due to too many changes.

**Pull request closed without merging**